spec/module.dd

```diff
@@ -407,6 +403,22 @@ void main()
 
 $(P $(D static) cannot be used with selective imports.)
 
+$(P Selective `import`s can be made from multiple modules if
+all `import`s are selective, or when the imported module has a qualified name:)
+
+$(SPEC_RUNNABLE_EXAMPLE_COMPILE
+---
+import std.stdio : writeln, std.file : write, std.ascii;
```

Comment:
What about an example that has multiple symbols per module? E.g.,
````
import std.stdio : writeln, writefln, std.file : write;
````

Reply:
(TBH I don't really like this syntax, because it mixes module names and symbols and is harder to read. But since it's in the spec, it should be illustrated properly.)

Reply:
Ok. done